src/components/forms/charcount.scss

```diff
@@ -2,7 +2,7 @@
 
 .char-count {
     letter-spacing: 1px;
-    color: lighten($type-gray, 30%);
+    color: $type-gray-75percent;
```

Comment:
This doesn't look the same, but that may have been intentional.

Before:
<img src="https://user-images.githubusercontent.com/3019167/43481444-f971179e-94d3-11e8-866a-d5c95f995759.png" width="300px" />

After:
<img src="https://user-images.githubusercontent.com/3019167/43481452-fd7dd200-94d3-11e8-89ad-0e4714a1492c.png" width="300px" />

Reply:
Thanks for calling this out, but this update is okay!